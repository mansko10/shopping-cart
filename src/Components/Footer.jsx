import StyledFooter from "./Styles/StyledFooter";
import { Github } from "lucide-react";

export default function Footer() {
  return (
    <>
      <StyledFooter className="footer">
        <a href="https://github.com/mansko10/shopping-cart" target="_blank">
          <Github /> View on Github
        </a>
      </StyledFooter>
    </>
  );
}
