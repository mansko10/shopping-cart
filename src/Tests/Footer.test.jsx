import Footer from "../Components/Footer";
import { expect, it } from "vitest";
import { render } from "@testing-library/react";

it("Renders Footer.jsx correctly", () => {
  const { container } = render(Footer);

  expect(container).toMatchSnapshot();
});
