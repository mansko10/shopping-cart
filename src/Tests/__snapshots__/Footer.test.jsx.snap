// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Renders Footer.jsx correctly 1`] = `<div />`;
